libs/daemon/internal/parent_process/server.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
Why mix Go and Python for this test?

You could build a single binary that acts both as the parent and the child process.

Reply:
There is no particular reason; either approach would work fine.